### ISSUE TITLE
Multiline Text Field

### ISSUE THREAD

**Issue by novachemtr** [KorzhCom/EasyData#54]
Hello, I would like to use the library however I did not see an attribute or definition for textarea or htmleditor type fields, could you please let me know if there is any or how we can implement? 

Thank you.

**Comment by antifree** [comment]
Hi,

It is not supported yet, but I think we can add it in the nearest update

**Comment by antifree** [comment]
Hello, we have published version 1.3.3 PR #55.

You can enable multiline text editor in such a way: 
```c#

 endpoints.MapEasyData((options) => {
    .     .      .     .    .
    options.UseModelTuner(model =>
    {
        // Find attribute by id (EntityType.PropertyName pattern)
        var categoryDesc = model.FindEntityAttr("Category.Description");
        // set default editor for its attribute
        categoryDesc.DefaultEditor = new TextValueEditor($"TVE_MULTI_{categoryDesc.Id}")
        {
            Multiline = true
        };
    });
});
```
P.s. There is a small problem with styles for the textarea. We will resolve it in the next update
